src/main/java/com/google/testing/compile/CompilationRule.java

```diff
@@ -56,7 +57,8 @@ public Statement apply(final Statement base, Description description) {
     return new Statement() {
       @Override public void evaluate() throws Throwable {
         final AtomicReference<Throwable> thrown = new AtomicReference<Throwable>();
-        Result result = Compilation.compile(ImmutableList.of(new AbstractProcessor() {
+        Result result = Compilation.compile(ToolProvider.getSystemJavaCompiler(),
```

Comment:
Would be even better if you somehow merged #71 into that one wrt passing a `JavaCompiler` to the `CompilationRule` rather than always using `ToolProvider.getSystemJavaCompiler()`. Sure one can easily be rebased on the other anyway…